src/pip/_vendor/vendor.txt

```diff
@@ -10,7 +10,7 @@ pep517==0.10.0
 progress==1.5
 pyparsing==2.4.7
 requests==2.25.1
-    certifi==2020.12.05
+    certifi==2020.12.5
```

Comment:
This seems wrong...

Reply:
I have a vague recollection that this happened in the 21.0 release, too...

Reply:
Fixed in vendoring 0.3.4.

It contains https://github.com/pradyunsg/vendoring/commit/d70bc6ec7fa73cc416a32c609a9792d0223e62c3.

Reply:
Ok, I've removed that commit.